fish_prompt.fish

```diff
@@ -420,7 +421,8 @@ function __bobthefish_prompt_status -S -a last_status -d 'Display flags for a no
     and set superuser 1
 
     # Jobs display
-    jobs -p >/dev/null
+    set num_bg_jobs (jobs -p | wc -l)
```

Comment:
it would be great if we didn't pay the extra forking cost when the config is disabled. something like...

```fish
if [ "$theme_display_jobs_verbose" = 'yes' ]
  set bg_jobs (jobs -p | wc -l)
  [ "$bg_jobs" -eq 0 ]
  and set bg_jobs # clear it out so it doesn't show when `0`
else
  jobs -p >/dev/null
  and set bg_jobs 1
end
```